connectors/github_mcp_connector.py

```
@@ -16,736 +16,730 @@
 from connectors.mcp_base import MCPConnector
 
 logger = logging.getLogger(__name__)
 
+
 class GitHubMCPConnector(MCPConnector):
     """
     GitHub MCP Connector for real repository access and integration
     """
-    
+
     def __init__(self, api_token: Optional[str] = None):
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)